app/assets/stylesheets/application.scss

```diff
@@ -299,7 +299,7 @@ ul.no_bullet_list li {
 }
 
 .display_none {
-  display: none;
+  display: none !important;
```

Comment:
There is a problem with this line, apparently Capybara (library for testing code) doesn't recognise elements that have `display: none !important`. Are you sure we need `!important` property here?